preload.js

```diff
@@ -31418,21 +31485,21 @@ function renderFormFromSettings(bookSettings) {
             .removeAttribute('selected');
     }
 
-    if (bookSettings.lockratio) {
-        document
-            .querySelector('option[value="lockratio"]')
-            .setAttribute('selected', '');
-        document
-            .querySelector('option[value="stretch"]')
-            .removeAttribute('selected');
-    } else {
-        document
-            .querySelector('option[value="stretch"]')
-            .setAttribute('selected', '');
-        document
-            .querySelector('option[value="lockratio"]')
-            .removeAttribute('selected');
-    }
+    // if (bookSettings.lockratio) {
```

Comment:
oh! I clipped these because I realized it kept losing my old setting and defaulting to the same one every time regardless of previous usage.... felt like it was fighting the browser. Is it ok if I leave this chopped? haven't noticed a functionality loss but might be looking in wrong spot